che-editors.yaml

```diff
@@ -287,10 +287,16 @@ editors:
       - name: che-machine-exec
         container:
           image: 'quay.io/eclipse/che-machine-exec:7.50.0'
+          memoryLimit: 128Mi
+          memoryRequest: 32Mi
+          cpuLimit: 500m
+          cpuRequest: 30m
           command:
             - /go/bin/che-machine-exec
             - '--url'
             - '127.0.0.1:3333'
+            - '--idle-timeout'
+            - '15m'
```

Comment:
If we set idling via the command line then we won't be able to specify it via the `CheCluster` (parmeters have precedence over the env variable) no?

Reply:
well, I believe we set it already for theia for 3.0 and nothing is going to change for 3.1 I think
for 3.2 we can remove those parameters once all the dedicated idling PRs are merged.

Reply:
I mean, when I opened [the idling issue](https://github.com/eclipse/che/issues/21390) I mentioned that 

> Theia definition should not specify the --idle-timeout parameter for machine exec.

Now we can temporarily set it for `stable` for consistency but as soon as @dkwon17 PRs get merged we should remove it for both `next` and `stable`.

Reply:
Yes, in the machine-exec idling PR: https://github.com/eclipse-che/che-machine-exec/pull/208, the parameters have precedence over the environment variable: https://github.com/eclipse-che/che-machine-exec/blob/aa76e6080e188822ab9c60abce6c01791befc3c7/cfg/cfg.go#L87-L97

Reply:
so, does these changes make any sense or it's better to wait when David's PRs will be merged?